Application/EdFi.Ods.Common/Security/Claims/EdFiResourceClaimValue.cs

```diff
@@ -54,12 +54,12 @@ public EdFiResourceClaimValue(string action, List<int> educationOrganizationIds)
         /// </summary>
         /// <param name="action">The action URI representing the action that the claim is authorized to perform on the resource.</param>
         /// <param name="educationOrganizationIds">The Local Education Agency Ids to which the claim applies.</param>
```

Comment:
We should look for and change the use of "Local Education Agency" in these sorts of comments and XML documentation and whenever we see them, update them to "Education Organization" to reflect the evolved capabilities of the authorization layer.

Reply:
I took care of this.